xkcdpass/xkcd_password.py

```diff
@@ -147,11 +147,10 @@ def wordlist_to_worddict(wordlist):
 
     # Maybe should be a defaultdict, but this reduces dependencies
     for word in wordlist:
-        try:
-            worddict[word[0]].append(word)
-        except KeyError:
-            worddict[word[0]] = [word, ]
-
+        if word[0] not in worddict:              # replaced try-catch with if statement
```

Comment:
This is a similar implementation to `collections.defaultdict`, and should be far more performant than testing every key: the exception will only be thrown and caught for the first word with a new starting letter, rather than testing every loop